nakama/boot.bat

```diff
@@ -1,2 +1,2 @@
 @ECHO off
-docker-compose -f docker-compose.yml up
+docker-compose -f %~dp0docker-compose.yml up
```

Comment:
Did you mean to rename this file?

Reply:
This is to look for docker-compose.yml on the same director as you ran the .bat file
I encountered an error when i tried to ran it in my E: drive.
Hope this would help out for other people and save you time answering issues.

Reply:
I'll let @Razoric480 test the Windows script, as I'm not on Windows.

Reply:
quoted from nakama docs https://github.com/heroiclabs/nakama-docs/blob/master/docs/install-docker-quickstart.md#running-nakama-with-docker-compose
> If you are trying to run Nakama via Docker-Compose on Windows, you'll need to make a small change to the downloaded docker-compose.yml file. Follow this instruction to bind the correct path.
If logging output does not immediately appear in stdout add tty: true to the nakama service in your docker-compose.yml file.

but apparently it only affects some people but no harm having it in

Reply:
The script is good; I hadn't originally written it with execution safety in mind, as it was mostly a shortcut to work with.